packages/communication/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@celo/communication",
```

Comment:
I said this above but how about `@celo/connect`?

Reply:
changed to `@celo/connect` :)